controllers/rcj_soccer_referee_supervisor/recorder/recorder.py

```diff
@@ -0,0 +1,70 @@
+from controller import Supervisor
+
+from pathlib import Path
+import datetime
+import time
+
+
+class VideoRecordAssistant:
+
+    def __init__(self,
+                 supervisor: Supervisor,
+                 output_path: str = "",
+                 fastforward_rate: int = 1,
+                 resolution: str = "720p"):
+
+        self.supervisor = supervisor
+        self.output_path = output_path
+        self.fastforward_rate = fastforward_rate
+        self.resolution = resolution
+
+        self.__recording = False
```

Comment:
Why `self.__recording` and not `self.recording`

Reply:
That's to prevent modification from out of class.